cubi_tk/archive/copy.py

```diff
@@ -198,11 +258,14 @@ def _restore_relative_symlinks(self, root, rel_symlinks, add_dangling=True):
             The symlink should be there, even though the target file is not accessible
             in the archived copy.
         """
-        for (relative, destination) in rel_symlinks:
-            if add_dangling or os.path.exists(
-                os.path.join(root, os.path.dirname(relative), destination)
-            ):
-                os.symlink(destination, os.path.join(root, relative))
+        for (relative, target) in rel_symlinks:
+            symlink_path = os.path.join(root, relative)
+            if os.path.exists(symlink_path) or os.path.islink(symlink_path):
+                continue
+            symlink_dir = os.path.dirname(symlink_path)
+            if add_dangling or os.path.exists(os.path.join(symlink_dir, target)):
+                os.makedirs(symlink_dir, mode=488, exist_ok=True)
```

Comment:
mode 488 could have a comment explaining that it means permissions 750 in octal notation.